packages/editor/src/toolbar/popups/color-picker.tsx

```diff
@@ -130,30 +138,47 @@ export function ColorPicker(props: ColorPickerProps) {
             onTouchEnd={() => onChange(currentColor)}
             onMouseUp={() => onChange(currentColor)}
           />
-          <Input
-            variant={"clean"}
-            placeholder="#000000"
-            spellCheck={false}
-            sx={{
-              my: 2,
-              p: 0,
-              borderRadius: 0,
-              fontSize: ["title", "title", "body"],
-              color: "fontTertiary",
-              textAlign: "center",
-              letterSpacing: 1.5
-            }}
-            value={currentColor.toUpperCase()}
-            maxLength={7}
-            onChange={(e) => {
-              const { value } = e.target;
-              if (!value) return;
-              if (tinycolor(value, { format: "hex" }).isValid()) {
-                setCurrentColor(value);
-                onChange(value);
-              }
-            }}
-          />
+          <Flex sx={{ flexDirection: "row" }}>
+            <Input
+              variant={"clean"}
+              placeholder="#000000"
+              spellCheck={false}
+              sx={{
+                my: 2,
+                p: 0,
+                borderRadius: 0,
+                fontSize: ["title", "title", "body"],
+                color: "fontTertiary",
+                textAlign: "center",
+                letterSpacing: 1.5
+              }}
+              value={currentColor.toUpperCase()}
+              maxLength={7}
+              onChange={(e) => {
+                const { value } = e.target;
+                if (!value) return;
+                if (tinycolor(value, { format: "hex" }).isValid()) {
+                  setCurrentColor(value);
+                  onChange(value);
+                }
+              }}
+            />
+            <Button
+              sx={{
+                width: "50%",
+                flexShrink: 0,
+                p: 0,
+                fontSize: ["title", "title", "body"],
+                fontWeight: 800,
+                color: `${currentColor}`,
```

Comment:
Using `currentColor` as the foreground is undesired since it'll become unreadable for low contrast colors (e.g. white/yellow). We should use a fixed color instead.